Vostok.Hosting/Setup/IVostokMetricsBuilderExtensions.cs

```diff
@@ -10,5 +12,12 @@ public static class IVostokMetricsBuilderExtensions
         /// </summary>
         public static IVostokMetricsBuilder SetupHerculesMetricEventSender([NotNull] this IVostokMetricsBuilder builder) =>
             builder.SetupHerculesMetricEventSender(b => b.Enable());
+
+        /// <inheritdoc cref="IVostokMetricsBuilder.EnrichAnnotationTags" />
+        public static IVostokMetricsBuilder EnrichAnnotationTags([NotNull] this IVostokMetricsBuilder builder, [NotNull] params (string key, string value)[] tags)
```

Comment:
I think it's better for now to just have EnrichInstanceAnnotationTags, which will only transform tags if instance tag exists (because we don't know other useful scenarios).